app/models/course/achievement.rb

```diff
@@ -0,0 +1,6 @@
+class Course::Achievement < ActiveRecord::Base
+  stampable
+
+  belongs_to :creator, class_name: User.name
+  belongs_to :course, inverse_of: :achievements
```

Comment:
might want to say that achievements are ordered by weights by default, as a `default_scope`.